spec/mspec-opal/runner.rb

```diff
@@ -14,7 +24,7 @@ def register
 
   def ===(description)
     @seen << description
-    @filters.include? description
+    @filters.filtered?(description)
```

Comment:
Why don't just put an "if ENV" here and stick to Set?

Same goes for the formatter, I'd just wrap the formatter assignment with "if ENV" and assign the inverted formatter instead. 

A $global instead of ENV would be ok too. 

As non-DRY as it is, it's always easy to do a project-wide search for the ENV var and find all the places in which it's used. Especially since this is basically internal configuration code (v. library code).